insights/parsers/rhv_log_collector_analyzer.py

```diff
@@ -0,0 +1,22 @@
+"""
+RHV Log Collector Analyzer
+=================================
+
```

Comment:
In order to include this parser in the documentation, please add a new file `docs/shared_parsers_catalog/rhv_log_collector_analyzer.rst` which looks like this:
```
.. automodule:: insights.parsers.rhv_log_collector_analyzer
   :members:
   :show-inheritance:
```
You can find information on writing and testing documentation here: https://insights-core.readthedocs.io/en/latest/docs_guidelines.html

I believe when you attempt to build the docs you will get an error from Sphinx caused by the `===...` on line 3 being longer than the title string above it.  They must be the same length.

Reply:
good catch, done